### ISSUE TITLE
Unable to reproduce fine-tuning results

### ISSUE THREAD

**Issue by netw0rkf10w** [facebookresearch/SWAG#9]
Hello,

I've tried reproducing the reported fine-tuning for RegNetY-16GF but obtained **only 50% top-1 accuracy** on ImageNet-1K at the end of training. I used the following hyper-parameters, taken from the paper:

```
# lr: 6e-3
# lr schedule: cosine
# batch size: 512
# weight-decay: 0
# resolution: 384x384
# training steps: 20000 ~= 8 epochs
# mixup: 0.1
# sync batchnorm
# EMA: decay 1e-4
```

I have some doubt regarding the number of training steps, which seems to be too low. 20K training steps is equivalent to 8 epochs:

`num_epochs = batch_size x num_steps / num_images = 512 x 20000/1281167 ~= 8`

Could you please confirm if this was indeed your setting?

Thank you in advance!

**Comment by mannatsingh** [comment]
@netw0rkf10w 50% sounds really bad which tells me that there might be a major bug in your workflow. The epochs calculation is correct and we indeed used 8 epochs for fine tuning that model. See https://github.com/facebookresearch/SWAG/issues/8#issuecomment-1131080339 for an example config for `RegNetY 128GF`. The recipe for `RegNetY 16GF` should also look the same - you can just increase the local batch size and use fewer GPUs.

**Comment by mannatsingh** [comment]
Closing this out since it seems like a bug in the fine tuning setup.
